src/routes/(app)/_components/layout/navbar.svelte

```diff
@@ -9,9 +18,17 @@
     </a>
   </div>
   <div class="flex justify-end gap-7">
-    <a href="#/">Equipa</a>
-    <a href="#/">Projetos</a>
-    <a href="#/">Eventos</a>
-    <a href="#/">Contactos</a>
+    <a class:active={currentUrl === "http://localhost:3000/#/team/"} href="#/team/"><p class = "font-bold">Equipa</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/projects/"} href="#/projects/"><p class = "font-bold">Projetos</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/events/"} href="#/events/"><p class = "font-bold">Eventos</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/contacts/"} href="#/contacts/"><p class = "font-bold">Contactos</p></a>
   </div>
 </nav>
+
+<style>
+  .active {
+    background-color: rgba(116, 20, 20, 0.4);
+    padding: 2px 3px; 
```

Comment:
A little more padding on the left and right would look good :) Also for it to be more readable you can use `theme('colors.muted-red.400')` (example), as to use the colors we already have defined, if you don't know what they are check out `tailwind.config.cjs`

Reply:
Done!